check-clang-tidy issues in master

It was mentioned in Slack that there are check-clang-tidy issues in master. CI doesn't show them, but if you comment out the lines in [run_clang_tidy_extra.py](https://github.com/cmu-db/terrier/blob/master/build-support/run_clang_tidy_extra.py#L34) that skip files based on not having a diff against master and just check every file, you get these:
```
 Checking: /git/terrier/test/execution/sql_vector_operations_vector_math_test.cpp
/git/terrier/test/execution/util_chunked_vector_test.cpp:150:12: error: invalid case style for function 'id' [readability-identifier-naming,-warnings-as-errors]
  uint32_t id() const noexcept { return id_; }
           ^~
           Id
 Checking: /git/terrier/test/execution/ast_test.cpp
/git/terrier/test/execution/sql_thread_state_container_test.cpp:34:14: error: invalid case style for member 'x' [readability-identifier-naming,-warnings-as-errors]
    uint64_t x{0};
             ^
             x_
/git/terrier/test/execution/sql_thread_state_container_test.cpp:35:14: error: invalid case style for member 'arr' [readability-identifier-naming,-warnings-as-errors]
    uint32_t arr[10] = {0};
             ^~~
             arr_
/git/terrier/test/execution/sql_thread_state_container_test.cpp:36:14: error: invalid case style for member 'arr_2' [readability-identifier-naming,-warnings-as-errors]
    uint32_t arr_2[2] = {44, 23};
             ^~~~~
             arr_2_
/git/terrier/test/execution/sql_thread_state_container_test.cpp:37:13: error: invalid case style for member 'next' [readability-identifier-naming,-warnings-as-errors]
    Object *next{nullptr};
            ^~~~
            next_
/git/terrier/test/execution/sql_thread_state_container_test.cpp:38:10: error: invalid case style for member 'initialized' [readability-identifier-naming,-warnings-as-errors]
    bool initialized{false};
         ^~~~~~~~~~~
         initialized_
 Checking: /git/terrier/test/execution/parsing_scanner_test.cpp
/git/terrier/test/execution/sql_sorter_test.cpp:161:12: error: invalid case style for member 'key' [readability-identifier-naming,-warnings-as-errors]
  uint32_t key;
           ^~~
           key_
/git/terrier/test/execution/sql_sorter_test.cpp:162:12: error: invalid case style for member 'data' [readability-identifier-naming,-warnings-as-errors]
  uint32_t data[N];
           ^~~~
           data_
 Checking: /git/terrier/test/execution/ast_context_test.cpp
/git/terrier/test/execution/parsing_scanner_test.cpp:54:21: error: invalid case style for member 'source' [readability-identifier-naming,-warnings-as-errors]
  const std::string source;
                    ^~~~~~
                    source_
/git/terrier/test/execution/parsing_scanner_test.cpp:55:28: error: invalid case style for member 'expected_tokens' [readability-identifier-naming,-warnings-as-errors]
  std::vector<Token::Type> expected_tokens;
                           ^~~~~~~~~~~~~~~
                           expected_tokens_
/git/terrier/test/execution/parsing_scanner_test.cpp:56:61: error: invalid case style for member 'check' [readability-identifier-naming,-warnings-as-errors]
  std::function<void(Scanner *scanner, uint32_t token_idx)> check;
                                                            ^~~~~
                                                            check_
 Checking: /git/terrier/util/execution/gen_opt_bc.cpp
/git/terrier/benchmark/runner/mini_runners.cpp:551:13: error: use emplace_back instead of push_back [modernize-use-emplace,-warnings-as-errors]
      param.push_back(parser::ConstantValueExpression(type_param, execution::sql::Integer(low_key)));
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                            ~
            emplace_back(
/git/terrier/benchmark/runner/mini_runners.cpp:555:13: error: use emplace_back instead of push_back [modernize-use-emplace,-warnings-as-errors]
      param.push_back(parser::ConstantValueExpression(type_param, execution::sql::Integer(low_key)));
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                            ~
            emplace_back(
/git/terrier/benchmark/runner/mini_runners.cpp:556:13: error: use emplace_back instead of push_back [modernize-use-emplace,-warnings-as-errors]
      param.push_back(parser::ConstantValueExpression(type_param, execution::sql::Integer(high_key)));
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                             ~
            emplace_back(
/git/terrier/benchmark/runner/mini_runners.cpp:1077:21: error: loop variable is copied but only used as const reference; consider making it a const reference [performance-for-range-copy,-warnings-as-errors]
          for (auto row : r) {
                    ^
               const  &
/git/terrier/benchmark/runner/mini_runners.cpp:1265:26: error: implicit conversion 'long long' -> bool [readability-implicit-bool-conversion,-warnings-as-errors]
    HandleBuildDropIndex(is_build, num_rows, key_num, type);
                         ^
                                  != 0
/git/terrier/benchmark/runner/mini_runners.cpp:1302:10: error: use emplace_back instead of push_back [modernize-use-emplace,-warnings-as-errors]
  params.push_back(parser::ConstantValueExpression(type, execution::sql::Integer(0)));
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                ~
         emplace_back(
/git/terrier/benchmark/runner/mini_runners.cpp:1305:12: error: use emplace_back instead of push_back [modernize-use-emplace,-warnings-as-errors]
    params.push_back(parser::ConstantValueExpression(type, execution::sql::Integer(0)));
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                ~
           emplace_back(
/git/terrier/benchmark/runner/mini_runners.cpp:1339:26: error: implicit conversion 'long long' -> bool [readability-implicit-bool-conversion,-warnings-as-errors]
    HandleBuildDropIndex(is_build, inner, key_num, type);
                         ^
                                  != 0
/git/terrier/benchmark/runner/mini_runners.cpp:1383:8: error: invalid case style for variable 'outerTbl' [readability-identifier-naming,-warnings-as-errors]
  auto outerTbl = execution::sql::TableGenerator::GenerateTableIndexName(type, outer);
       ^~~~~~~~
       outer_tbl
/git/terrier/benchmark/runner/mini_runners.cpp:1384:8: error: invalid case style for variable 'innerTbl' [readability-identifier-naming,-warnings-as-errors]
  auto innerTbl = execution::sql::TableGenerator::GenerateTableIndexName(type, inner);
       ^~~~~~~~
       inner_tbl
/git/terrier/benchmark/runner/mini_runners.cpp:1533:26: error: implicit conversion 'long long' -> bool [readability-implicit-bool-conversion,-warnings-as-errors]
    HandleBuildDropIndex(is_build, row, num_integers + num_decimals, type);
                         ^
                                  != 0
/git/terrier/benchmark/runner/mini_runners.cpp:1585:10: error: use emplace_back instead of push_back [modernize-use-emplace,-warnings-as-errors]
  params.push_back(parser::ConstantValueExpression(type, execution::sql::Integer(0)));
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                ~
         emplace_back(
/git/terrier/benchmark/runner/mini_runners.cpp:1588:12: error: use emplace_back instead of push_back [modernize-use-emplace,-warnings-as-errors]
    params.push_back(parser::ConstantValueExpression(type, execution::sql::Integer(0)));
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                ~
           emplace_back(
/git/terrier/benchmark/runner/mini_runners.cpp:1635:26: error: implicit conversion 'long long' -> bool [readability-implicit-bool-conversion,-warnings-as-errors]
    HandleBuildDropIndex(is_build, row, num_integers + num_decimals, type);
                         ^
                                  != 0
/git/terrier/benchmark/runner/mini_runners.cpp:1667:10: error: use emplace_back instead of push_back [modernize-use-emplace,-warnings-as-errors]
  params.push_back(parser::ConstantValueExpression(type, execution::sql::Integer(0)));
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                ~
         emplace_back(
/git/terrier/benchmark/runner/mini_runners.cpp:1670:12: error: use emplace_back instead of push_back [modernize-use-emplace,-warnings-as-errors]
    params.push_back(parser::ConstantValueExpression(type, execution::sql::Integer(0)));
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                ~
           emplace_back(
/git/terrier/benchmark/runner/mini_runners.cpp:2141:13: error: loop variable is copied but only used as const reference; consider making it a const reference [performance-for-range-copy,-warnings-as-errors]
  for (auto title : titles) {
            ^
       const  &
/git/terrier/benchmark/runner/mini_runners.cpp:2165:15: error: invalid case style for member 'match' [readability-identifier-naming,-warnings-as-errors]
  const char *match;
              ^~~~~
              match_
/git/terrier/benchmark/runner/mini_runners.cpp:2166:8: error: invalid case style for member 'found' [readability-identifier-naming,-warnings-as-errors]
  bool found;
       ^~~~~
       found_
/git/terrier/benchmark/runner/mini_runners.cpp:2167:15: error: invalid case style for member 'value' [readability-identifier-naming,-warnings-as-errors]
  const char *value;
              ^~~~~
              value_
/git/terrier/benchmark/runner/mini_runners.cpp:2168:7: error: invalid case style for member 'intValue' [readability-identifier-naming,-warnings-as-errors]
  int intValue;
      ^~~~~~~~
      int_value_
The files that failed were:
[u'/git/terrier/test/execution/util_chunked_vector_test.cpp',
 u'/git/terrier/test/execution/sql_thread_state_container_test.cpp',
 u'/git/terrier/test/execution/sql_sorter_test.cpp',
 u'/git/terrier/test/execution/parsing_scanner_test.cpp',
 u'/git/terrier/benchmark/runner/mini_runners.cpp']
Note that a failing .h file will fail all the .cpp files that include it.
```
This issue is strictly related to fixing the bugs in master, not fixing the script itself.

--- a/benchmark/runner/mini_runners.cpp
+++ b/benchmark/runner/mini_runners.cpp
@@ -1062,9 +1062,9 @@ void NetworkQueriesOutputRunners(pqxx::work *txn) {
 
           if (col > 1) {
             query_ss << ",";
-            for (int i = 1; i < col; i++) {
+            for (int j = 1; j < col; j++) {
               query_ss << "nprunnersdummy" << type_s << "()";
-              if (i != col - 1) {
+              if (j != col - 1) {
                 query_ss << ",";
               }
             }
@@ -1074,9 +1074,9 @@ void NetworkQueriesOutputRunners(pqxx::work *txn) {
           pqxx::result r{txn->exec(query_ss.str())};
 
           // Get all the results
-          for (const auto &row : r) {
-            for (auto i = 0; i < col; i++) {
-              null << row[i];
+          for (const auto &result_row : r) {
+            for (auto j = 0; j < col; j++) {
+              null << result_row[j];
             }
           }
         }
@@ -2182,31 +2182,32 @@ int main(int argc, char **argv) {
 
   for (int i = 0; i < argc; i++) {
     for (auto *arg : args) {
-      if (strstr(argv[i], arg->match) != nullptr) {
-        arg->found = true;
-        arg->value = strstr(argv[i], "=") + 1;
-        arg->intValue = atoi(arg->value);
+      if (strstr(argv[i], arg->match_) != nullptr) {
+        arg->found_ = true;
+        arg->value_ = strstr(argv[i], "=") + 1;
+        arg->int_value_ = atoi(arg->value_);
       }
     }
   }
 
-  if (port_info.found) terrier::runner::port = port_info.intValue;
-  if (skip_large_rows_runs_info.found) terrier::runner::skip_large_rows_runs = true;
-  if (warm_num_info.found) terrier::runner::warmup_iterations_num = warm_num_info.intValue;
-  if (rerun_info.found) terrier::runner::rerun_iterations = rerun_info.intValue;
-  if (updel_info.found) terrier::runner::updel_limit = updel_info.intValue;
-  if (warm_limit_info.found) terrier::runner::warmup_rows_limit = warm_limit_info.intValue;
+  if (port_info.found_) terrier::runner::port = port_info.int_value_;
+  if (skip_large_rows_runs_info.found_) terrier::runner::skip_large_rows_runs = true;
+  if (warm_num_info.found_) terrier::runner::warmup_iterations_num = warm_num_info.int_value_;
+  if (rerun_info.found_) terrier::runner::rerun_iterations = rerun_info.int_value_;
+  if (updel_info.found_) terrier::runner::updel_limit = updel_info.int_value_;
+  if (warm_limit_info.found_) terrier::runner::warmup_rows_limit = warm_limit_info.int_value_;
 
   terrier::LoggersUtil::Initialize();
   SETTINGS_LOG_INFO("Starting mini-runners with this parameter set:");
-  SETTINGS_LOG_INFO("Port ({}): {}", port_info.match, terrier::runner::port);
-  SETTINGS_LOG_INFO("Skip Large Rows ({}): {}", skip_large_rows_runs_info.match, terrier::runner::skip_large_rows_runs);
-  SETTINGS_LOG_INFO("Warmup Iterations ({}): {}", warm_num_info.match, terrier::runner::warmup_iterations_num);
-  SETTINGS_LOG_INFO("Rerun Iterations ({}): {}", rerun_info.match, terrier::runner::rerun_iterations);
-  SETTINGS_LOG_INFO("Update/Delete Index Limit ({}): {}", updel_info.match, terrier::runner::updel_limit);
-  SETTINGS_LOG_INFO("Warmup Rows Limit ({}): {}", warm_limit_info.match, terrier::runner::warmup_rows_limit);
-  SETTINGS_LOG_INFO("Filter ({}): {}", filter_info.match, filter_info.value);
-  SETTINGS_LOG_INFO("Compiled ({}): {}", compiled_info.match, compiled_info.found);
+  SETTINGS_LOG_INFO("Port ({}): {}", port_info.match_, terrier::runner::port);
+  SETTINGS_LOG_INFO("Skip Large Rows ({}): {}", skip_large_rows_runs_info.match_,
+                    terrier::runner::skip_large_rows_runs);
+  SETTINGS_LOG_INFO("Warmup Iterations ({}): {}", warm_num_info.match_, terrier::runner::warmup_iterations_num);
+  SETTINGS_LOG_INFO("Rerun Iterations ({}): {}", rerun_info.match_, terrier::runner::rerun_iterations);
+  SETTINGS_LOG_INFO("Update/Delete Index Limit ({}): {}", updel_info.match_, terrier::runner::updel_limit);
+  SETTINGS_LOG_INFO("Warmup Rows Limit ({}): {}", warm_limit_info.match_, terrier::runner::warmup_rows_limit);
+  SETTINGS_LOG_INFO("Filter ({}): {}", filter_info.match_, filter_info.value_);
+  SETTINGS_LOG_INFO("Compiled ({}): {}", compiled_info.match_, compiled_info.found_);
 
   // Benchmark Config Environment Variables
   // Check whether we are being passed environment variables to override configuration parameter
@@ -2219,8 +2220,8 @@ int main(int argc, char **argv) {
 
   terrier::runner::InitializeRunnersState();
 
-  if (filter_info.found) {
-    if (compiled_info.found) {
+  if (filter_info.found_) {
+    if (compiled_info.found_) {
       terrier::runner::MiniRunners::mode = terrier::execution::vm::ExecutionMode::Compiled;
     }
 

--- a/benchmark/runner/mini_runners.cpp
+++ b/benchmark/runner/mini_runners.cpp
@@ -548,12 +548,12 @@ static void GenIdxScanParameters(type::TypeId type_param, int64_t num_rows, int6
 
     std::vector<parser::ConstantValueExpression> param;
     if (lookup_size == 1) {
-      param.push_back(parser::ConstantValueExpression(type_param, execution::sql::Integer(low_key)));
+      param.emplace_back(type_param, execution::sql::Integer(low_key));
       bounds.emplace_back(low_key, low_key);
     } else {
       auto high_key = low_key + lookup_size - 1;
-      param.push_back(parser::ConstantValueExpression(type_param, execution::sql::Integer(low_key)));
-      param.push_back(parser::ConstantValueExpression(type_param, execution::sql::Integer(high_key)));
+      param.emplace_back(type_param, execution::sql::Integer(low_key));
+      param.emplace_back(type_param, execution::sql::Integer(high_key));
       bounds.emplace_back(low_key, high_key);
     }
 
@@ -1074,7 +1074,7 @@ void NetworkQueriesOutputRunners(pqxx::work *txn) {
           pqxx::result r{txn->exec(query_ss.str())};
 
           // Get all the results
-          for (auto row : r) {
+          for (const auto &row : r) {
             for (auto i = 0; i < col; i++) {
               null << row[i];
             }
@@ -1262,7 +1262,7 @@ BENCHMARK_DEFINE_F(MiniRunners, SEQ2_0_IndexScanRunners)(benchmark::State &state
       throw "Invalid is_build argument for IndexScan";
     }
 
-    HandleBuildDropIndex(is_build, num_rows, key_num, type);
+    HandleBuildDropIndex(is_build != 0, num_rows, key_num, type);
     return;
   }
 
@@ -1299,10 +1299,10 @@ BENCHMARK_DEFINE_F(MiniRunners, SEQ2_0_IndexScanRunners)(benchmark::State &state
 
   std::vector<parser::ConstantValueExpression> params;
   std::vector<type::TypeId> param_types;
-  params.push_back(parser::ConstantValueExpression(type, execution::sql::Integer(0)));
+  params.emplace_back(type, execution::sql::Integer(0));
   param_types.push_back(type);
   if (lookup_size > 1) {
-    params.push_back(parser::ConstantValueExpression(type, execution::sql::Integer(0)));
+    params.emplace_back(type, execution::sql::Integer(0));
     param_types.push_back(type);
   }
 
@@ -1336,7 +1336,7 @@ BENCHMARK_DEFINE_F(MiniRunners, SEQ2_1_IndexJoinRunners)(benchmark::State &state
       throw "Invalid is_build argument for IndexJoin";
     }
 
-    HandleBuildDropIndex(is_build, inner, key_num, type);
+    HandleBuildDropIndex(is_build != 0, inner, key_num, type);
     return;
   }
 
@@ -1380,12 +1380,12 @@ BENCHMARK_DEFINE_F(MiniRunners, SEQ2_1_IndexJoinRunners)(benchmark::State &state
     predicate = preds.str();
   }
 
-  auto outerTbl = execution::sql::TableGenerator::GenerateTableIndexName(type, outer);
-  auto innerTbl = execution::sql::TableGenerator::GenerateTableIndexName(type, inner);
+  auto outer_tbl = execution::sql::TableGenerator::GenerateTableIndexName(type, outer);
+  auto inner_tbl = execution::sql::TableGenerator::GenerateTableIndexName(type, inner);
 
   std::stringstream query;
-  query << "SELECT " << cols << " FROM " << outerTbl << " AS a, " << innerTbl << " AS b WHERE " << predicate;
-  auto f = std::bind(&MiniRunners::IndexNLJoinChecker, this, innerTbl, key_num, std::placeholders::_1,
+  query << "SELECT " << cols << " FROM " << outer_tbl << " AS a, " << inner_tbl << " AS b WHERE " << predicate;
+  auto f = std::bind(&MiniRunners::IndexNLJoinChecker, this, inner_tbl, key_num, std::placeholders::_1,
                      std::placeholders::_2);
   auto equery = OptimizeSqlStatement(query.str(), std::make_unique<optimizer::TrivialCostModel>(), std::move(units), f);
   BenchmarkExecQuery(num_iters, equery.first.get(), equery.second.get(), true);
@@ -1530,7 +1530,7 @@ void MiniRunners::ExecuteUpdate(benchmark::State *state) {
       throw "Invalid is_build argument for ExecuteUpdate";
     }
 
-    HandleBuildDropIndex(is_build, row, num_integers + num_decimals, type);
+    HandleBuildDropIndex(is_build != 0, row, num_integers + num_decimals, type);
     return;
   }
 
@@ -1582,10 +1582,10 @@ void MiniRunners::ExecuteUpdate(benchmark::State *state) {
 
   std::vector<parser::ConstantValueExpression> params;
   std::vector<type::TypeId> param_types;
-  params.push_back(parser::ConstantValueExpression(type, execution::sql::Integer(0)));
+  params.emplace_back(type, execution::sql::Integer(0));
   param_types.push_back(type);
   if (car > 1) {
-    params.push_back(parser::ConstantValueExpression(type, execution::sql::Integer(0)));
+    params.emplace_back(type, execution::sql::Integer(0));
     param_types.push_back(type);
   }
 
@@ -1632,7 +1632,7 @@ void MiniRunners::ExecuteDelete(benchmark::State *state) {
       throw "Invalid is_build argument for ExecuteDelete";
     }
 
-    HandleBuildDropIndex(is_build, row, num_integers + num_decimals, type);
+    HandleBuildDropIndex(is_build != 0, row, num_integers + num_decimals, type);
     return;
   }
 
@@ -1664,10 +1664,10 @@ void MiniRunners::ExecuteDelete(benchmark::State *state) {
 
   std::vector<parser::ConstantValueExpression> params;
   std::vector<type::TypeId> param_types;
-  params.push_back(parser::ConstantValueExpression(type, execution::sql::Integer(0)));
+  params.emplace_back(type, execution::sql::Integer(0));
   param_types.push_back(type);
   if (car > 1) {
-    params.push_back(parser::ConstantValueExpression(type, execution::sql::Integer(0)));
+    params.emplace_back(type, execution::sql::Integer(0));
     param_types.push_back(type);
   }
 
@@ -2138,7 +2138,7 @@ void RunMiniRunners() {
   // Do post-processing
   std::vector<std::string> titles = {"OUTPUT", "SCANS",  "IDX_SCANS", "SORTS", "HJ",
                                      "AGGS",   "INSERT", "UPDATE",    "DELETE"};
-  for (auto title : titles) {
+  for (const auto &title : titles) {
     char target[64];
     snprintf(target, sizeof(target), "execution_%s.csv", title.c_str());
 
@@ -2162,10 +2162,10 @@ void RunMiniRunners() {
 }
 
 struct Arg {
-  const char *match;
-  bool found;
-  const char *value;
-  int intValue;
+  const char *match_;
+  bool found_;
+  const char *value_;
+  int int_value_;
 };
 
 int main(int argc, char **argv) {

--- a/test/execution/parsing_scanner_test.cpp
+++ b/test/execution/parsing_scanner_test.cpp
@@ -73,7 +73,7 @@ void CheckEquality(uint32_t test_idx, const std::vector<Token::Type> &expected,
 void RunTests(const std::vector<TestCase> &tests) {
   for (unsigned test_idx = 0; test_idx < tests.size(); test_idx++) {
     const auto &test = tests[test_idx];
-    Scanner scanner(test.source.data(), test.source.length());
+    Scanner scanner(test.source_.data(), test.source_.length());
 
     std::vector<Token::Type> actual;
 
@@ -81,13 +81,13 @@ void RunTests(const std::vector<TestCase> &tests) {
     for (auto token = scanner.Next(); token != Token::Type::EOS; token = scanner.Next(), token_idx++) {
       actual.push_back(token);
 
-      if (test.check != nullptr) {
-        test.check(&scanner, token_idx);
+      if (test.check_ != nullptr) {
+        test.check_(&scanner, token_idx);
       }
     }
 
     // Expect final sizes should be the same
-    CheckEquality(test_idx, test.expected_tokens, actual);
+    CheckEquality(test_idx, test.expected_tokens_, actual);
   }
 }
 

--- a/test/execution/parsing_scanner_test.cpp
+++ b/test/execution/parsing_scanner_test.cpp
@@ -51,9 +51,9 @@ TEST_F(ScannerTest, SimpleSourceTest) {
 namespace {
 
 struct TestCase {
-  const std::string source;
-  std::vector<Token::Type> expected_tokens;
-  std::function<void(Scanner *scanner, uint32_t token_idx)> check;
+  const std::string source_;
+  std::vector<Token::Type> expected_tokens_;
+  std::function<void(Scanner *scanner, uint32_t token_idx)> check_;
 };
 
 }  // namespace

--- a/test/execution/sql_sorter_test.cpp
+++ b/test/execution/sql_sorter_test.cpp
@@ -161,7 +161,7 @@ struct TestTuple {
   uint32_t key_;
   uint32_t data_[N];
 
-  int32_t Compare(const TestTuple<N> &other) const { return key - other.key; }
+  int32_t Compare(const TestTuple<N> &other) const { return key_ - other.key_; }
 };
 
 // Generic function to perform a parallel sort. The input parameter indicates the sizes of each
@@ -199,7 +199,7 @@ void TestParallelSort(exec::ExecutionContext *exec_ctx, const std::vector<uint32
     auto *sorter = container.AccessCurrentThreadStateAs<Sorter>();
     for (uint32_t i = 0; i < sorter_sizes[tid]; i++) {
       auto *elem = reinterpret_cast<TestTuple<N> *>(sorter->AllocInputTuple());
-      elem->key = mt() % 3333;
+      elem->key_ = mt() % 3333;
     }
   });
 

--- a/test/execution/sql_sorter_test.cpp
+++ b/test/execution/sql_sorter_test.cpp
@@ -158,8 +158,8 @@ TEST_F(SorterTest, TopKTest) {
 
 template <uint32_t N>
 struct TestTuple {
-  uint32_t key;
-  uint32_t data[N];
+  uint32_t key_;
+  uint32_t data_[N];
 
   int32_t Compare(const TestTuple<N> &other) const { return key - other.key; }
 };

--- a/test/execution/sql_thread_state_container_test.cpp
+++ b/test/execution/sql_thread_state_container_test.cpp
@@ -46,17 +46,17 @@ TEST_F(ThreadStateContainerTest, ComplexObjectContainerTest) {
       [](UNUSED_ATTRIBUTE auto *_, auto *s) {
         // Set some stuff to indicate object is initialized
         auto obj = new (s) Object();
-        obj->x = 10;
-        obj->initialized = true;
+        obj->x_ = 10;
+        obj->initialized_ = true;
       },
       nullptr, nullptr);
   ForceCreationOfThreadStates(&container, 4);
 
   // Check
   container.ForEach<Object>([](Object *obj) {
-    EXPECT_EQ(10u, obj->x);
-    EXPECT_EQ(nullptr, obj->next);
-    EXPECT_EQ(true, obj->initialized);
+    EXPECT_EQ(10u, obj->x_);
+    EXPECT_EQ(nullptr, obj->next_);
+    EXPECT_EQ(true, obj->initialized_);
   });
   EXECUTION_LOG_TRACE("{} thread states", container.GetThreadStateCount());
 }

--- a/test/execution/sql_thread_state_container_test.cpp
+++ b/test/execution/sql_thread_state_container_test.cpp
@@ -31,11 +31,11 @@ TEST_F(ThreadStateContainerTest, EmptyStateTest) {
 // NOLINTNEXTLINE
 TEST_F(ThreadStateContainerTest, ComplexObjectContainerTest) {
   struct Object {
-    uint64_t x{0};
-    uint32_t arr[10] = {0};
-    uint32_t arr_2[2] = {44, 23};
-    Object *next{nullptr};
-    bool initialized{false};
+    uint64_t x_{0};
+    uint32_t arr_[10] = {0};
+    uint32_t arr_2_[2] = {44, 23};
+    Object *next_{nullptr};
+    bool initialized_{false};
   };
 
   MemoryPool memory(nullptr);

--- a/test/execution/util_chunked_vector_test.cpp
+++ b/test/execution/util_chunked_vector_test.cpp
@@ -147,7 +147,7 @@ class Simple {
 
   ~Simple() { count--; }
 
-  uint32_t id() const noexcept { return id_; }
+  uint32_t Id() const noexcept { return id_; }
 
  private:
   uint32_t id_;

--- a/test/execution/util_chunked_vector_test.cpp
+++ b/test/execution/util_chunked_vector_test.cpp
@@ -164,9 +164,9 @@ TEST_F(ChunkedVectorTest, ClearTest) {
   v.emplace_back(3);
 
   EXPECT_EQ(3u, v.size());
-  EXPECT_EQ(1u, v[0].id());
-  EXPECT_EQ(2u, v[1].id());
-  EXPECT_EQ(3u, v[2].id());
+  EXPECT_EQ(1u, v[0].Id());
+  EXPECT_EQ(2u, v[1].Id());
+  EXPECT_EQ(3u, v[2].Id());
 
   v.clear();
 
@@ -177,9 +177,9 @@ TEST_F(ChunkedVectorTest, ClearTest) {
   v.emplace_back(12);
 
   EXPECT_EQ(3u, v.size());
-  EXPECT_EQ(10u, v[0].id());
-  EXPECT_EQ(11u, v[1].id());
-  EXPECT_EQ(12u, v[2].id());
+  EXPECT_EQ(10u, v[0].Id());
+  EXPECT_EQ(11u, v[1].Id());
+  EXPECT_EQ(12u, v[2].Id());
 }
 
 // NOLINTNEXTLINE

